config/locales/en.yml

```diff
@@ -680,15 +680,16 @@ en:
   page_title:
     login: Login - Track-a-query
     forgot_password: Forgot your password - Track-a-query
-    change_password: Change your password - Track-a-query
+    change_password: Forgot your password - Track-a-query
```

Comment:
This should be "Change your password". This is the page the user sees after clicking on the "password" reset link in the "Forgot password" email